### DISCUSSION TITLE
Can I remove all the files inside the var\cache\pacman\pkg folder?

### DISCUSSION THREAD

**Discussion by asmwarrior** [msys2/MSYS2-packages#4430] [Q&A] [answered]
Hi, I found that the folder:

`<msys2_installed_folder\var\cache\pacman\pkg`

has many zst files, I just see the whole folder is about 12G byte. The oldest file I see is from year 2017.

So, can I remove all the files in this folder?

Thanks.

**Comment by Biswa96** [comment, marked as answer]
yes, you can remove them with `pacman -Scc` command.

**Reply by asmwarrior** [reply]
OK, thanks for the reply.

When I asked this question, I have searched the keyword "cache" in the issue or in the discussions before, but I can't find a suitable solution.

So, this answer may also help others in the future.

**Reply by Biswa96** [reply]
That command is similar as `apt autoclean`. This page may help https://wiki.archlinux.org/title/Pacman/Rosetta

**Comment by asmwarrior** [comment]
> That command is similar as `apt autoclean`. This page may help https://wiki.archlinux.org/title/Pacman/Rosetta

OK, thanks for the information.
